### ISSUE TITLE
Bug on Kraken, can't find order

### ISSUE THREAD

**Issue by thierryjmartin** [freqtrade/freqtrade#4278]
## Describe your environment

  * Operating system: Docker
  * Python Version: 3.8
  * CCXT version: _____ (`pip freeze | grep ccxt`)
  * Freqtrade Version: 2020.12
  
Note: All issues other than enhancement requests will be closed without further comment if the above template is deleted or not filled out.

## Describe the problem:

*Explain the problem you have encountered*

### Steps to reproduce:

  1. Run the bot and wait until kraken API breakdown
  
### Observed Results:

  * freqtrade proccess were killed

### Relevant code exceptions or logs

Note: Please copy/paste text of the messages, no screenshots of logs please.

  ```
2021-01-30 00:07:02,978 - freqtrade.commands.trade_commands - ERROR - 'OOVSLN-ESBH3-DLBZQY'
2021-01-30 00:07:02,978 - freqtrade.commands.trade_commands - ERROR - Fatal exception!
Traceback (most recent call last):
  File "/freqtrade/freqtrade/commands/trade_commands.py", line 19, in start_trading
    worker.run()
  File "/freqtrade/freqtrade/worker.py", line 74, in run
    state = self._worker(old_state=state)
  File "/freqtrade/freqtrade/worker.py", line 111, in _worker
    self._throttle(func=self._process_running, throttle_secs=self._throttle_secs)
  File "/freqtrade/freqtrade/worker.py", line 132, in _throttle
    result = func(*args, **kwargs)
  File "/freqtrade/freqtrade/worker.py", line 145, in _process_running
    self.freqtrade.process()
  File "/freqtrade/freqtrade/freqtradebot.py", line 176, in process
    self.check_handle_timedout()
  File "/freqtrade/freqtrade/freqtradebot.py", line 1067, in check_handle_timedout
    order = self.exchange.fetch_order(trade.open_order_id, trade.pair)
  File "/freqtrade/freqtrade/exchange/common.py", line 135, in wrapper
    return f(*args, **kwargs)
  File "/freqtrade/freqtrade/exchange/exchange.py", line 1074, in fetch_order
    return self._api.fetch_order(order_id, pair)
  File "/root/.local/lib/python3.8/site-packages/ccxt/kraken.py", line 1146, in fetch_order
    order = self.parse_order(self.extend({'id': id}, orders[id]))
KeyError: 'OOVSLN-ESBH3-DLBZQY'
```

**Comment by xmatthias** [comment]
Please fill out the issue template, or i'll have to close this issue as it's unknown which version you're using.

**Comment by xmatthias** [comment]
Please upgrade to the lastest version (that's why it's important to upgrade to the latest version).

It's been a bug in ccxt, that was fixed 20some days ago. 
It'll be fixed in 2021.1 or develop.

**Comment by thierryjmartin** [comment]
Thanks !

Freqtrade is great !
